bot.py

```diff
@@ -1,154 +1,157 @@
-    if task["data"]["completed"]:
-        new_status = False
-    else:
-        new_status = True
-    client.query(q.update(q.ref(q.collection("todo"), todo_id),
-                          {"data": {"completed": new_status}}))
-    context.bot.send_message(
-        chat_id=chat_id, text="Successfully updated todo task status 👌\n\nSee all your todo with /list_todo")
-
-
-def delete_todo(update, context):
-    chat_id = update.effective_chat.id
-    message = update.message.text
-    todo_id = message.split("_")[1]
-
-    client.query(q.delete(q.ref(q.collection("todo"), todo_id)))
-    context.bot.send_message(
-        chat_id=chat_id, text="Successfully deleted todo task status 👌\n\nAdd a new todo with /add_todo")
-
-""" To print the and track the last cmd's """
-def echo(update, context):
-    chat_id = update.effective_chat.id
-    message = update.message.text
-
-    user = client.query(q.get(q.match(q.index("users"), chat_id)))
-    last_command = user["data"]["last_command"]
-
-    if last_command == "add_todo":
-        todo = client.query(q.create(q.collection("todo"), {
-            "data": {
-                "user_id": chat_id,
-                "todo": message,
-                "completed": False,
-                "date": datetime.now(pytz.UTC)
-            }
-        }))
-        client.query(q.update(q.ref(q.collection("users"), user["ref"].id()), {
-                     "data": {"last_command": ""}}))
-        context.bot.send_message(
-            chat_id=chat_id, text="Successfully added todo task 👍\n\nSee all your todo with /list_todo")
-
-""" Handling all the above request respone sys using dispatcher handeler """
-dispatcher.add_handler(CommandHandler("start", start))
-dispatcher.add_handler(CommandHandler("add_todo", add_todo))
-dispatcher.add_handler(CommandHandler("list_todo", list_todo))
-dispatcher.add_handler(MessageHandler(
-    Filters.regex("/update_[0-9]*"), update_todo))
-dispatcher.add_handler(MessageHandler(
-    Filters.regex("/delete_[0-9]*"), delete_todo))
-dispatcher.add_handler(MessageHandler(Filters.text, echo))
-
-""" updater.start_polling() """ 
-
-updater.start_webhook(listen="0.0.0.0",
-                      port=int(PORT),
-                      url_path=telegram_bot_token,
-                      )
-updater.bot.setWebhook('https://faunadbbot.herokuapp.com/'+telegram_bot_token)
-
-# Run the bot until you press Ctrl-C or the process receives SIGINT,
-# SIGTERM or SIGABRT. This should be used most of the time, since
-# start_polling() is non-blocking and will stop the bot gracefully.
+""" Importing the modules for timestaps, telegram handeler, and faunadb """
+
+import pytz
+import telegram
+from datetime import datetime
+from telegram.ext import Updater
+from telegram.ext import CommandHandler
+from telegram.ext import MessageHandler, Filters
+from faunadb import query as q
+from faunadb.objects import Ref
+from faunadb.client import FaunaClient
+import os
+
+import config
```

Comment:
This won't help. The code fails at lines 34 and 36 due to non-existent names `telegram_bot_token` and `faunt_secret`.

Use the following instead:
``` py
from config import telegram_bot_token, fauna_secret
```